deploy_packages.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+import sys
+import os
+import shutil
+from shutil import copyfile
+
+deployable_packages = ['common', 'datamodel', 'parsers', 'gl_extensions', 'input_manager', 'renderer', 'ticker', 'traits', 'window_manager', 'camera'] # left for later use
+
+def usage():
+    print("deploy_packages.py")
+    print("A simple python script for engine deb packages build and deployment.")
+    print('Usage:')
+    print('python3 ./deploy_packages.py root/directory/with/built/packages [bazel-bin] required-engine-version [0.0.3] target-architecture [amd64] resulting-tar-path [/tmp/deploy.tar.gz]')
+
+def clear_dir(path):
+    for filename in os.listdir(path):
+        file_path = os.path.join(path, filename)
+        try:
+            if os.path.isfile(file_path) or os.path.islink(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path):
+                shutil.rmtree(file_path)
+        except Exception as e:
+            print('Failed to delete %s. Reason: %s' % (file_path, e))
+
+def clear_and_rm_dir(tmp_dir):
+    try:
+        clear_dir(tmp_dir)
+        os.rmdir(tmp_dir)
+    except OSError:
```

Comment:
Consider not catching it here and let the caller decide

Reply:
No need to do this, if it can't clean up so be it. It shouldn't crash the deploy. The files are stored inside `/tmp` anyway

Reply:
'Kay